src/axom/primal/tests/primal_intersect.cpp

```diff
@@ -1073,7 +1073,7 @@ TEST( primal_intersect, 3D_triangle_triangle_intersection_regression )
   {
     // https://github.com/LLNL/axom/issues/152
     std::string msg = "From Axom github issue #152 (simplified)";
-    Point3 vdata[] = { Point3::make_point( 0.066,  0,  0.2133).
+    Point3 vdata[] = { Point3::make_point( 0.066,  0,  0.2133),
```

Comment:
I found the bug in my test -- I had a period (`.`) instead of a comma (`,`) in this array initializer list.

Interestingly, the `toss3` tests all passed, but the `blueos` tests all failed in the previous version.